md/two-main-types-of-deployment.md

```diff
@@ -50,7 +45,150 @@ But you can still use the pre packaged Tomcat bundles or the `bonita.war` webapp
+set ENGINE_OPTS=%ENGINE_OPTS% "-Dorg.bonitasoft.engine.api-type.application.name=bonita"
+set ENGINE_OPTS=%ENGINE_OPTS% "-Dorg.bonitasoft.platform.username=platformAdmin" "-Dorg.bonitasoft.platform.password=platform"
+
+set CATALINA_OPTS=%CATALINA_OPTS% %ENGINE_OPTS% %PLATFORM_SETUP% ...
+```
+
+Remove the `<bonita-installation-directory>/server/webapps/bonita` folder if it exists
+Edit the `WEB-INF/web.xml` file in the `<bonita-installation-directory>/server/webapps/bonita.war` file and comment the
+following lines to disable Engine resources
+
+Community Edition only: listener starting the Engine
+```xml
+<listener> 
+    <listener-class>org.bonitasoft.engine.api.internal.servlet.EngineInitializerListener</listener-class> 
+</listener>
+```
+
+Subscription Edition only: listener starting the Engine
+```xml
+<listener> 
+    <listener-class>com.bonitasoft.engine.api.internal.servlet.EngineInitializerListenerSP</listener-class> 
+</listener>
+```
+
+HTTP Engine API Configuration
+```xml
+    <servlet>
+        <servlet-name>HttpAPIServlet</servlet-name>
+        <servlet-class>org.bonitasoft.engine.api.internal.servlet.HttpAPIServlet</servlet-class>
+    </servlet>
+...
+    <servlet-mapping>
+        <servlet-name>HttpAPIServlet</servlet-name>
+        <url-pattern>/serverAPI/*</url-pattern>
+    </servlet-mapping>
+```
+
+Disable XA datasources managed by Bitronix by commenting or deleting the following line in the `<bonita-installation-directory>/server/conf/server.xml` file 
+```xml
+  <Listener className="bitronix.tm.integration.tomcat55.BTMLifecycleListener" />
+```
+
+Disable datasources managed by Tomcat by commenting or removing database resources declared in the in the `<bonita-installation-directory>/setup/tomcat-templates/bonita.xml` file
+
+
+##### Wildfly
+
+Edit the `<bonita-installation-directory>/setup/wildfly-templates/standalone.xml` file as described in the following
+
+Configure the Engine Client
+```xml
+    <system-properties>
+        ...
+        <!-- Engine Client configuration -->
+        <property name="org.bonitasoft.engine.api-type" value="HTTP" />
+        <property name="org.bonitasoft.engine.api-type.server.url" value="http://localhost:8080" />
+        <property name="org.bonitasoft.engine.api-type.application.name" value="bonita" />
+        <property name="org.bonitasoft.platform.username" value="platformAdmin" />
+        <property name="org.bonitasoft.platform.password" value="platform" />
+    </system-properties>
+```
+
+Remove or comment the datasources configuration
+```xml
+    <subsystem xmlns="urn:jboss:domain:datasources:4.0">
+        <datasources>
+        ...
+        </datasources>
+    </subsystem>
+```
+
+
+Edit the `<bonita-installation-directory>/server/standalone/deployments/bonita-all-in-one-x.y.z.ear/bonita.war/WEB-INF/web.xml`
+
+Apply the same changes as for the Tomcat bundle to disable Engine resources
+
+
```

Comment:
In the "Configuring the Bonita Stack" chapter there's no mention to changes in the server with the engine. Is it correct?

Reply:
Yes, that's correct
I will add a section about this to make this explicit